source/uwp/SharedRenderer/lib/XamlHelpers.cpp

```diff
@@ -242,11 +243,25 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering::XamlHelpers
         return content;
     }
 
-    winrt::Image CreateBackgroundImage(winrt::AdaptiveRenderContext const& renderContext, winrt::hstring const& url)
+    winrt::Image CreateBackgroundImage(winrt::AdaptiveRenderContext const& renderContext, winrt::hstring const& url, winrt::TileControl const& tileControl)
     {
         try
         {
             auto imageUrl = GetUrlFromString(renderContext.HostConfig(), url);
+            auto imagePath = HStringToUTF8(imageUrl.Path());
+            bool isImageSvg = imagePath.find("svg") != std::string::npos;
+
+            if (isImageSvg)
+            {
+                winrt::SvgImageSource svgImageSource{};
+
+                ConfigureSvgImageSourceAsync(renderContext, url, imageUrl, svgImageSource, tileControl);
+
+                winrt::Image backgroundImage;
+                backgroundImage.Source(svgImageSource);
+
+                return backgroundImage;
+			}
```

Comment:
nit: indentation

Reply:
Same comment as above, this shows up fine in visual studio.

![image](https://github.com/microsoft/AdaptiveCards/assets/165726045/b2f306b9-b2d6-444e-a5c8-874e77d8454d)